Writerside/topics/raptor.md

```diff
@@ -203,4 +166,64 @@ In the context of the RAPTOR algorithm, a **Route** refers to:
 - **Optimization**: Routes are scanned efficiently, with the algorithm looking at each route at most once per round,
   contributing to RAPTOR's speed and dynamic capabilities.
 
-![raptor-route.png](raptor-route.png)
+![raptor-route.png](raptor-route.png){width="650"}
+
+## The RAPTOR Algorithm Explained in Detail
+
+### Key Features of RAPTOR
+
+**Pareto-Optimality:** RAPTOR computes Pareto-optimal journeys, meaning it considers multiple criteria, such as earliest
+arrival time and the fewest transfers. A journey is Pareto-optimal if no other journey is better in both criteria.
```

Comment:
This is incorrect, a journey can be considered pareto-optimal if no objective (num transfers or arrival time) can be improved without worsening the other.

Reply:
I would call it Connection and not Journey. Since we defined Journey as realized Connections. Connections are Alternatives to choose from.

Reply:
**Pareto-Optimality:** RAPTOR computes Pareto-optimal connections, meaning it considers multiple criteria, such as
earliest arrival time and the fewest transfers. A connection is Pareto-optimal if no objective (number of transfers or
arrival time) can be improved without worsening the other.